dashboards/clowder-dashboard.yml

```diff
@@ -194,7 +194,7 @@ data:
           "steppedLine": false,
           "targets": [
             {
-              "expr": "sum by (method) (rate(rest_client_requests_total{namespace=\"clowder-system\"}[1m]))",
+              "expr": "sum by (method) (rate(rest_client_requests_total[1m]))",
```

Comment:
Why dropping the namespace - this doesn't seem right? Without it we will be pulling metrics from any container that has the same metric exposed.